addons/event_booth_sale/models/sale_order.py

```diff
@@ -23,6 +24,13 @@ def _compute_event_booth_count(self):
     def action_confirm(self):
         res = super(SaleOrder, self).action_confirm()
         for so in self:
+            so_lines_missing_booth = self.env['sale.order.line']
+            for line in so.order_line:
+                if line.product_type == 'event_booth' and not line.event_booth_pending_ids:
+                    so_lines_missing_booth |= line
+            if so_lines_missing_booth:
+                event_names = "".join(["- " + event.name + "\n" for event in so_lines_missing_booth])
+                raise ValidationError(_("Please make sure all your event-related lines are configured before confirming this order: \n%s" % event_names))
```

Comment:
So we want the same error as for events? We do not have a booth-related message?

Reply:
Yes, I discussed that with LNA during BE testing stage and he came up with the wordings for the errors